fhir-quality-measure/src/main/java/com/ibm/fhir/ecqm/common/package-info.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/**
+ * This package provides model-independent classes for evaluating FHIR clinical
+ * quality measures. This code was ported from the cql-evaluator/evaluator.measure
+ * module and adapted to remove an unnecessary dependency on the HAPI FHIR model
+ * and to support supplemental data elements as seen in the cqf-ruler project.
+ * 
+ * @see https://github.com/DBCG/cql-evaluator
+ * @see https://github.com/DBCG/cqf-ruler
+ */
```

Comment:
has this previously been cleared to be ported?

Reply:
We have cleared the cqf-ruler, but not the cql-evaluator. It should go through review.